tpm2/src/unseal/main.c

```diff
@@ -151,6 +167,12 @@ int main(int argc, char **argv)
     {
       kmyth_log(LOG_ERR, "kmyth-unseal encountered invalid outfile path");
       free(default_outPath);
+      kmyth_clear_and_free(outputData, outputSize);
```

Comment:
Like in getkey, after the call to tpm2_kmyth_unseal(), the authString and ownerAuthPasswd variables are no longer needed. Therefore, we could clear them after the if statement surrounding the tpm2_kmyth_unseal() call (lines 141-154). Then we wouldn't have to worry about them after that.